src/Pages/About.tsx

```diff
@@ -78,20 +88,20 @@ const About = ({ language, urlSlug, t }) => {
 
         const currentLanguage = language || defaultLanguage;
 
-        setMetadata(data => ({
+        setMetadata((data: any) => ({
```

Comment:
Found a souple of these.

Reply:
[9ede5d4](https://github.com/Kentico/kontent-sample-app-react/pull/206/commits/9ede5d4e8127366824a28d497840cf0cf5e1cc2e)